includes/ucf-post-list-config.php

```diff
@@ -111,6 +112,15 @@ public static function get_options() {
 					'default' => 'Search'
 				) ),
 				'list_title'          => new UCF_Post_List_Option( 'list_title' ),
+				'show_excerpt'		  => new UCF_Post_List_Option( 'list_excerpt', array (
+					'default'		  => false,
+					'format_callback' => array( 'UCF_Post_List_Config', 'format_option_bool')
+				) ),
+				'max_words'			  => new UCF_Post_List_Option( 'max_words', array (
```

Comment:
From a naming perspective, I think `excerpt_length` might be a little more clear here vs `max_words`.